pkg/network/node/pod.go

```diff
@@ -165,7 +169,8 @@ func getIPAMConfig(clusterNetworks []common.ClusterNetwork, localSubnet string)
 // Start the CNI server and start processing requests from it
 func (m *podManager) Start(socketPath string, localSubnetCIDR string, clusterNetworks []common.ClusterNetwork) error {
 	if m.enableHostports {
-		m.hostportSyncer = kubehostport.NewHostportSyncer()
+		iptInterface := utiliptables.New(utilexec.New(), utildbus.New(), utiliptables.ProtocolIpv4)
```

Comment:
is this a boring change? cc @openshift/sig-networking

Reply:
Mostly boring, yes. We want to eventually use only a single iptables.Interface object in pkg/network/node, but we're not there right now.